src/open_inwoner/pdc/models/product.py

```diff
@@ -201,6 +202,10 @@ def get_absolute_url(self, category=None):
     def has_cta_tag(self):
         return "\[CTABUTTON\]" in self.content
 
+    def save(self, *args, **kwargs):
+        self.content = html.escape(self.content).replace("\\&", "&")
+        super().save()
+
```

Comment:
As discussed in person: we established the field stores markdown syntax, but markdown allows HTML tags.

I though about it again: I think the html-escape should happen later, at render-time just before we render the markdown (eg: not here in save like we talked about at some point). What we could do here is `strip_tags()` to get rid of html tags, and leave the entities to the later escape.

This way the code is still editable (users `&`'s won't be` &amp;`'s). Possibly the markdown render function should be updated to do this.

This leaves the mystery of figuring out where the random slashes come from, and if they also show up back in the editor.